app/src/main/res/layout/language_check_item.xml

```diff
@@ -50,11 +50,11 @@
     android:layout_height="wrap_content"
     android:layout_alignParentRight="true"
     android:layout_centerVertical="true"
-    android:layout_marginRight="16dp"
-    android:layout_marginTop="6dp"
-    android:layout_marginBottom="6dp"
-    android:textSize="18sp"
+    android:layout_marginRight="@dimen/language_entries_count_margin_right"
+    android:layout_marginTop="@dimen/language_entries_count_margin_top"
+    android:layout_marginBottom="@dimen/language_entries_count_margin_bottom"
+    android:textSize="@dimen/language_entries_count_text_size"
     android:textColor="@color/grey"
-    tools:text="47"/>
+    tools:text="@string/language_entries_count_text"/>
```

Comment:
As far as I can see this isn't needed 47 was a random number and can be removed.

Reply:
Okay. Will rectify it.